clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11213,7 +11213,7 @@ def err_sycl_invalid_accessor_property_list_template_param : Error<
   "template parameter must be a "
   "%select{parameter pack|type|non-negative integer}1">;
 def err_sycl_num_kernel_wrong_reqd_wg_size : Error<
-  "%0 attribute must evenly divide the work-group size for the %1 attribute">;
+  "%0 attribute must evenly divide X argument of %1 attribute">;
```

Comment:
I think the original wording is better (esp because `X` and `Z` have that confusing swap behavior). I think a better approach is to change AttrDocs.td to spell out which argument is the work group size argument.

Reply:
Thanks @AaronBallman for the reviews. I have kept the original diagnostic message and updated AttrDocs.td to spell out which argument is the work group size argument for num_simd attribute.